tests/test_permission_to_partial_update_is_supported.py

```
@@ -0,0 +1,188 @@
+            ],
+        },
+    )
+
+    assert response.status_code == status.HTTP_200_OK, response.json()
+    author.refresh_from_db()
+    assert author.books.count() == 1
+    assert (b := author.books.first()) is not None and b.id == book.id
+
+
+@pytest.mark.django_db
+def test_partial_update_permitted_resource_with_unpermitted_related_resource_should_fail(
```

Ruff (0.8.2)

Line too long (89 > 88)

(E501)